source/whats-new.txt

```diff
@@ -21,6 +22,25 @@ Learn what's new in:
 * :ref:`Version 4.1 <version-4.1>`
 * :ref:`Version 4.0 <version-4.0>`
 
+.. _version-4.7.1:
+
+What's New in 4.7.1
+-------------------
+
+The 4.7.1 driver patches a bug that affects
+Client-Side Field Level Encryption and Queryable Encryption.
+The bug can cause data corruption when rotating :ref:`Data Encryption Keys <csfle-key-architecture>`
+(DEKs) encrypted with a :ref:`Customer Master Key <csfle-key-architecture>`
+hosted on Google Cloud Key Management Service or Azure
+Key Vault. The bug is present in the ``RewrapManyDataKey`` method
+and causes the loss of your DEKs.
```

Comment:
Issue: from this sentence, it is unclear whether the bug currently exists in 4.7.1.
Suggestion: use past tense and identify which version(s) the bug was present in.